### PR TITLE
Release 2.0

### LINKED ISSUE #14: Clean up CRW templates
- Consider deleting the intron templates
- Make sure each organism has only 1 template
- Are there any closely related templates where a single representative can be selected?

### LINKED ISSUE #38: Use RNArtist instead of R2R to avoid overlaps in Rfam templates
The idea is to run [RNArtist](https://github.com/fjossinet/RNArtistCore) on Rfam families, get [JSON output](https://raw.githubusercontent.com/fjossinet/RNArtist/master/RNArtist/samples/projects/1u6b.json), convert it into R2DT templates and then anytime anyone tries to visualise a sequence from that Rfam family, R2DT will draw it in the same layout as the consensus.

RNArtist layouts are superior to R2R because they have fewer overlaps (for example, 18S rRNA can be drawn without any overlaps).

### LINKED ISSUE #115: Speed up template selection using Ribovore + Blast
The ribovore team is working on a prototype.

### LINKED ISSUE #121: Possible off-by-one error?
Seems like in the `generate_traveler_fasta` function (linked below) that there's a possible off-by-one error in generated dot-bracket structures when a sequence contains a leading 5' label.

https://github.com/RNAcentral/R2DT/blob/main/utils/rna2djsonschema.py

The first for loop would ignore the 5' label when generating the nucleotide sequence but the base-pair indices used in the second for loop to generate the dot-bracket structure are not updated to match the now shorter sequence.

### LINKED ISSUE #124: Bundle Rfam seeds into cms tarball?
As part of the RNAcentral pipeline, we run R2DT in a nextflow process that makes use of a singularity container converted from the docker container built in this repo.

We currently set up for execution by downloading and expanding `cms.tar.gz` and bind-mounting the resulting folder to the correct place within the R2DT container. This worked in previous versions of R2DT (<1.3).

Now that R2DT can transfer pseudoknots, it needs access to the Rfam seed files. As these are not contained in the prepared data, R2DT attempts to download them. However, the directory `/rna/r2dt/data/rfam/` is not writeable, since it is within the singularity container and not bind-mounted.

Would it be possible to include the Rfam seeds in the precomputed library?

### PR DESCRIPTION
**Pull request by AntonPetrov**
- Streamline using custom templates (#129)
- GitHub Codespaces integration (#127)
- Consistent use of `--quiet` (#129)
- Bug fixes and docs improvements (fix #121)
- Rfam version 14.10 (https://github.com/RNAcentral/R2DT/pull/132)
- RNArtist templates (https://github.com/RNAcentral/R2DT/pull/140)
- Use Blast for template selection (https://github.com/RNAcentral/R2DT/pull/142)
- Fix #115
- Fix #124 
- Fix #14
- Fix #38
- Update version number to 2.0
- Scale up templates
- Update precomputed library

### CODE REVIEW

File: tests/tests.py [hunk 1]
```diff
@@ -93,6 +93,7 @@ def tearDown(self):
             print(f"Test results can be found in {self.test_results}")
         else:
             self.delete_folder(self.test_results)
+        runner.run(f"rm -f {os.path.join('examples', '*.ssi')}")
```
Comment (anayden) [inline tests/tests.py:96]: maybe avoid spawning a new process and use python api here?

```python
[os.remove(file) for file in glob.glob('examples/*.ssi')]
```

File: utils/gtrnadb.py [hunk 2]
```diff
@@ -42,6 +46,42 @@ def setup():
...
+    in between can be either.
+
+    Ser
+    >= 70 bp are usually SerTGA (with a D-arm).
+    < 65 are usually SerGCT (without a D-arm).
+    in between can be either.
+    """
+    if anticodon != "NNN":
+        return anticodon
+    seq_length = max(start, end) - min(start, end) + 1
```
Comment (anayden) [inline utils/gtrnadb.py:72]: minor: `abs(end - start) + 1` does the same and is a bit shorter

File: utils/gtrnadb.py [hunk 3]
```diff
@@ -176,7 +223,8 @@ def get_trnascan_cm(domain, isotype):
 
     result = runner.run(f"cmfetch -o {cm_output} {cm_library} {cm_name}")
     if result:
-        os.remove(cm_output)
+        if cm_output.exists():
+            cm_output.unlink()
```
Comment (anayden) [inline utils/gtrnadb.py:227]: no need for the `if` statement _if_ you use `cm_output.unlink(missing_ok=True)`

File: tests/tests.py (hunk 1 above)
Comment (AntonPetrov) [reply tests/tests.py]: Good point! I also [created an issue](https://github.com/RNAcentral/R2DT/issues/113) to go through the code and systematically reduce the number of system calls.

File: utils/gtrnadb.py (hunk 3 above)
Comment (AntonPetrov) [reply utils/gtrnadb.py]: You are right, thanks!

File: utils/gtrnadb.py (hunk 2 above)
Comment (AntonPetrov) [reply utils/gtrnadb.py]: Thank you very much. All suggested changes pushed now ✅

**Pull request closed without merging**